neo/Network/P2P/Payloads/StateRoot.cs

```diff
@@ -0,0 +1,110 @@
+using Neo.Cryptography;
+using Neo.IO;
+using Neo.IO.Json;
+using Neo.Persistence;
+using Neo.Plugins;
+using Neo.SmartContract;
+using Neo.Wallets;
+using System;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class StateRoot : IInventory
+    {
+        public byte Version;
+        public uint Index;
+        public UInt256 PreHash;
```

Comment:
It is not link to the block?

Reply:
`PreHash` is previous stateroot has.
`index` is the block index.

Reply:
We need both?